src/types/pools.ts

```diff
@@ -64,7 +64,8 @@ export type Pools = {
   };
   Factories: Record<string, FactoryType>;
   Stakable: {
-    AllowList: string[];
+    VotingAllowed: string[];
+    VotingDisabled: string[];
```

Comment:
This is confusing to me now. Why do we have these two attributes and why are VotingDisabled pools still Stakable? Either we need comments to explain the logic here or the naming could be better.

Reply:
Yeah, I still find the naming confusing. Maybe instead it could be something like:
```suggestion
    // Pools to be included in the voting gauges list.
    VotingGaugePools: string[];
    // Pools that have additional rewards but are not included in the VotingGaugePools list.
    AllowList: string[];
```

Reply:
Sounds good. Changed.